test/unit/control/SnapControl.spec.ts

```diff
@@ -282,6 +282,27 @@ describe("SnapControl", () => {
         expect(position).to.be.lessThan(flicking.panels[flicking.panelCount - 1].position);
         expect(position).to.equal(flicking.panels[0].position);
       });
+
+      it("Should move to the nearest panel from the camera, when animation is interrupted by user input", async () => {
+        const flicking = await createFlicking(El.DEFAULT_HORIZONTAL, {
+          moveType: MOVE_TYPE.SNAP,
+          align: "prev",
+        });
+
+        const control = flicking.control;
+        const camera = flicking.camera;
+
+        // Suppress animation interrupt error
+        const promise = flicking.moveTo(2, 1500).catch(error => error);
+        tick(500);
+        const position = camera.position;
+        // Simulate interrupt
+        await simulate(flicking.element, { deltaX: 0, duration: 100 }, 1000);
+        tick(1000);
+        await promise;
+
+        expect(control.activePanel.index).to.equal(camera.findNearestAnchor(position).panel.index);
```

Comment:
Can you add a `expect` that `control.activePanel.index` not equals `2`? To prove that it's correctly interrupted.

Reply:
(not in this test case, but...) Index can be 2 even when interruption occurs if camera position is very close to last panel... so I didn't add that.

Reply:
(not in this test case, but...) Index can be 2 even when interruption occurs if camera position is very close to last panel...
So checking if the fallback behavior was triggered seems more appropriate.

Reply:
That's weird, maybe the element's width is wrong, or something is wrong with `tick` in that case?
Please report to me for that case when it occurs again, I'll try to fix that.

Another approach could be listening to `move` event and then interrupting when the `nearestAnchor` is panel 1.
Anyway, I think it's fine for now 👌

Reply:
Isn't it normal behavior to do so? I mean, when the sliding almost ended and camera is closest to the destination slide (slide 2), the fallback should `moveTo` 2.

Reply:
The problem is, we're using sinon's fake timer, and constant time for `tick` and `simulate`.
The result should be consistent in that kind of environment. If not, there must be something that we're missing.
For example, there was a case where some tests were not removing elements after the test so that affected other test cases depending on the test execution order. We have to exclude such cases.

Reply:
Ah I see what you mean... there's no problem, the results are consistent. I meant just wrote that way for genereal cases.